client/blocks/inline-help/inline-help-search-results.jsx

```diff
@@ -43,6 +56,26 @@ function HelpSearchResults( {
 	placeholderLines,
 	track,
 } ) {
+	useEffect( () => {
+		// Cancel all queued speak messages.
+		loadingSpeak.cancel();
+		resultsSpeak.cancel();
+		errorSpeak.cancel();
+
+		// If there's no query, then we don't need to announce anything.
+		if ( isEmpty( searchQuery ) ) {
+			return;
+		}
+
+		if ( isSearching ) {
+			loadingSpeak();
+		} else if ( ! hasAPIResults ) {
+			errorSpeak();
+		} else if ( hasAPIResults ) {
```

Comment:
Perhaps it makes sense to alter this `else if ( hasAPIResults )` to just `else` because considering the above statement, `hasAPIResults` is guaranteed to be truthy?

Reply:
Good point! I could be misremembering, but here's the only reason I can think of as to why it would need to be like this:
- There's actually a "Default results" set at the beginning that we do not want to announce for. .We don't want to speak the message if it's going back to the default results. 
- You can test this state by searching for results, then clearing the search field. 
- We do not want to announce "Search results loaded" when clearing the search field, only if results have returned from the API related to the search.